### DISCUSSION TITLE
Some results for decoder on CUB

### DISCUSSION THREAD

**Discussion by mhh0318** [lucidrains/DALLE2-pytorch#128] [Show and tell]
I've trained a small model for the DALLE.2 Decoder part on CUB dataset.

output size is 64*64.

The sample process is the p_sample instead of DDIM. I would update the DDIM sample result later.

The major codes is from GLIDE and some of them are inspired by this repo.

 *Given Image for CLIP embedding*
![Given Image for CLIP embedding](https://user-images.githubusercontent.com/42776955/171073140-2fefa2c2-c496-40bc-b66f-198f45e1c1df.png )

*Generated Sample*
![Generated Sample](https://user-images.githubusercontent.com/42776955/171073139-362a47ef-0a37-4a42-9619-9184bccf36fa.png )

*Training Loss*
![training loss](https://user-images.githubusercontent.com/42776955/171074030-edc16877-03ea-42d6-9e32-f7e003fb4127.png)

**Comment by xinmiaolin** [comment]
Hi,

Thanks for sharing the information. Is it possible to know the following:

1- what preprocessing did you apply on the cub 200 dataset?
2- did you use image embeddings to the decoder?
3- the unet configurations in the decoder?

Thank you very much!
